moto/sagemaker/models.py

```diff
@@ -0,0 +1,364 @@
+            "ml.m5.xlarge",
+            "ml.m4.10xlarge",
+            "ml.t2.large",
+            "ml.m5.12xlarge",
+            "ml.m4.xlarge",
+            "ml.t3.large",
+            "ml.m5.24xlarge",
+            "ml.m4.2xlarge",
+            "ml.p2.8xlarge",
+            "ml.m5.2xlarge",
+            "ml.p3.8xlarge",
+            "ml.m4.4xlarge",
+        ]
+        if not validators.is_one_of(instance_type, VALID_INSTANCE_TYPES):
+            message = "Value '{}' at 'instanceType' failed to satisfy constraint: Member must satisfy enum value set: {}".format(
+                instance_type, VALID_INSTANCE_TYPES
+            )
+            raise RESTError(
+                error_type="ValidationException",
+                message=message,
+                template="error_json",
+            )
+
+    @property
+    def arn(self):
+        return (
+            "arn:aws:sagemaker:"
+            + self.region_name
+            + ":"
+            + str(ACCOUNT_ID)
+            + ":notebook-instance/"
+            + self.notebook_instance_name
+        )
+
+    @property
+    def url(self):
+        return "{}.notebook.{}.sagemaker.aws".format(
+            self.notebook_instance_name, self.region_name
+        )
+
+    def start(self):
+        self.status = "InService"
+
+    @property
+    def is_deletable(self):
+        return self.status in ["Stopped", "Failed"]
+
+    def stop(self):
+        self.status = "Stopped"
+
+
+class SageMakerModelBackend(BaseBackend):
+    def __init__(self, region_name=None):
+        self._models = {}
+        self.notebook_instances = {}
+        self.region_name = region_name
+
+    def reset(self):
+        region_name = self.region_name
+        self.__dict__ = {}
+        self.__init__(region_name)
+
+    def create_model(self, **kwargs):
+        model_obj = Model(
+            self.region_name,
+            kwargs.get("ModelName"),
+            kwargs.get("ExecutionRoleArn"),
+            kwargs.get("PrimaryContainer", {}),
+            kwargs.get("VpcConfig", {}),
+            kwargs.get("Containers", []),
+            kwargs.get("Tags", []),
+        )
+
+        self._models[kwargs.get("ModelName")] = model_obj
+        return model_obj.response_create
+
+    def describe_model(self, model_name=None):
+        for model in self._models.values():
+            # If a registry_id was supplied, ensure this repository matches
+            if model.ModelName != model_name:
+                continue
+            return model.response_object
+        message = "Could not find model '{}'.".format(
+            Model.arn_for_model_name(model_name, self.region_name)
+        )
+        raise RESTError(
+            error_type="ValidationException", message=message, template="error_json",
+        )
+
+    def list_models(self):
+        models = []
+        for model in self._models.values():
+            model_response = model.response_object.copy()
+            model_response.update(model.response_create)
```

Comment:
`response_create` adds the model_arn - isn't that already part of the model-class in the first place? 
In other words - why is this line necessary?

Reply:
Agreed.  Done.